packages/block-editor/src/hooks/dimensions.js

```diff
@@ -172,7 +172,13 @@ export function useCustomSides( blockName, feature ) {
 		return;
 	}
 
-	return support[ feature ];
+	if ( Array.isArray( support[ feature ] ) ) {
```

Comment:
Nice. Is this to catch axial values like `margin: (2) ['top', 'bottom']`?

Would it worth writing a small comment to that effect as a hint to others?

Reply:
Yep! That's the case: I've added an explanatory comment in https://github.com/WordPress/gutenberg/pull/41098/commits/e779a9ff8f52d550eee6bb2c147f30d10ea0c015 to hopefully make it clearer.